fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/physical/PhysicalDistributionOperator.java

```diff
@@ -34,16 +34,6 @@ public void setGlobalDicts(
         this.globalDicts = globalDicts;
     }
 
-    @Override
```

Comment:
Why remove these methods?

Reply:
In order to remove duplicate PhysicalDistributionOperator，we think they are equals only when they are same obj now.